templates/ci/xcode.yml

```diff
@@ -6,10 +6,14 @@
 pool:
   vmImage: 'macOS 10.13'
 
+variables:
+  Configuration: 'Release'
+  SDK: 'iphoneos'
+
 steps:
 - task: Xcode@5
   inputs:
+    actions: 'build'
+    xcWorkspacePath: '**/*.xcodeproj/project.xcworkspace'
```

Comment:
Seems odd to build without specifying a workspace, even if we have a good default.